### PR TITLE
[WIP] Aambroise add caption

### PR DESCRIPTION
**Pull request by aambroise-cartera**
##**Summary:**  
This PR highlights a change to the caption

##**Leading Questions**
@svalevski-cartera 

##**Next Steps**
No other steps required

### CODE REVIEW

**Comment by ssawe-cartera** [comment]
Your changes look very good! I would add "Flex Capacitor" and just ship it :)

**Review by ssawe-cartera — APPROVED** [review]
LGTM!

**Comment by aambroise-cartera** [comment]
Thank you sir

File: _posts/0000-01-17-aambroise-cartera.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is my 2nd caption change
+
```
Comment (ssawe-cartera) [inline _posts/0000-01-17-aambroise-cartera.md:7]:
```suggestion
third caption to test suggestion thingy
```

Comment (aambroise-cartera) [inline _posts/0000-01-17-aambroise-cartera.md:7]: This is a test

**Pull request closed without merging**